<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit</class>
 <widget class="QDialog" name="Deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>928</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
  background-image: url(:/Image/fone.png); 
}</string>
  </property>
  <widget class="QLabel" name="label_amount_profit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>840</y>
     <width>200</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rate_of_exchange">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color: white;
	border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>9.8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_with">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>540</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(43, 51, 63);
font: 20;

border: solid white;
border-width: 1px;
border-radius: 25px;
}

QPushButton:pressed {
  background-color:  rgb(53, 71, 84);
border: solid white;
border-width: 2px;
border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string> Добавить снятие</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>750</y>
     <width>230</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>СУММА НА ВКЛАДЕ К КОНЦУ СРОКА:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>810</y>
     <width>110</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>СУММА НАЛОГА:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="withdrawal_time">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>540</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="time">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>180</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>780</y>
     <width>180</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>НАЧИСЛЕННЫЕ ПРОЦЕНТЫ:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="amount_money">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color: white;
	border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>100000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>30</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>руб.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="withdrawal">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>540</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color: white;
	border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QComboBox" name="rate_of_payment">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="pop_time">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>460</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="time_replament">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color: white;
	border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_percent">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>780</y>
     <width>200</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pop_num">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>460</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color: white;
	border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>840</y>
     <width>120</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>СУММА ПРИБЫЛИ:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_money">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>750</y>
     <width>200</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_tax">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>810</y>
     <width>200</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>240</y>
     <width>20</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Капиталлизация процентов</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_menuCredit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>64</width>
     <height>62</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{
  font: 18pt &quot;.AppleSystemUIFont&quot;;
  border-image: url(:/Image/Menu1.png) stretch; 
}
QToolButton:pressed {
    border-image: url(:/Image/Button_nuber_after.png) stretch; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>2</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgb(51, 73, 90);
font: 20;

border: solid white;
border-width: 1px;
border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>СУММА ВКЛАДА</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_exitCredit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>64</width>
     <height>62</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{
  font: 18pt &quot;.AppleSystemUIFont&quot;;
  border-image: url(:/Image/Exit.png) stretch; 
}
QToolButton:pressed {
    border-image: url(:/Image/Button_nuber_after.png) stretch; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="pushButton_calculate_deposit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>630</y>
     <width>171</width>
     <height>54</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Bodoni 72 Smallcaps</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{
  border-image: url(:/Image/Game_Button_after.png) stretch; 
}
QToolButton:pressed {
    border-image: url(:/Image/Game_Button_befor.png) stretch; 
}</string>
   </property>
   <property name="text">
    <string>Результат</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>2</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgb(51, 73, 90);
font: 20;

border: solid white;
border-width: 1px;
border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>СРОК РАЗМЕЩЕНИЯ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>2</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgb(51, 73, 90);
font: 20;

border: solid white;
border-width: 1px;
border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>ПРОЦЕНТНАЯ СТАВКА</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>2</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgb(51, 73, 90);
font: 20;

border: solid white;
border-width: 1px;
border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>ПЕРИОДИЧНОСТЬ ВЫПЛАТ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>2</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgb(51, 73, 90);
font: 20;

border: solid white;
border-width: 1px;
border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>ПОПОЛНЕНИЯ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>530</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>2</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgb(51, 73, 90);
font: 20;

border: solid white;
border-width: 1px;
border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>ЧАСТИЧНЫЕ СНЯТИЯ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_pop">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>460</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(43, 51, 63);
font: 20;

border: solid white;
border-width: 1px;
border-radius: 25px;
}

QPushButton:pressed {
  background-color:  rgb(53, 71, 84);
border: solid white;
border-width: 2px;
border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string> Добавить снятие</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>740</y>
     <width>231</width>
     <height>131</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgb(70, 103, 123);
font: 20;

border: solid white;
border-width: 1px;
border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>740</y>
     <width>281</width>
     <height>131</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgb(51, 73, 90);
font: 20;

border: solid white;
border-width: 1px;
border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>370</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgb(51, 73, 90);
font: 20;

border: solid white;
border-width: 1px;
border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <zorder>label_24</zorder>
  <zorder>label_23</zorder>
  <zorder>label_22</zorder>
  <zorder>label_amount_profit</zorder>
  <zorder>rate_of_exchange</zorder>
  <zorder>Button_with</zorder>
  <zorder>label_11</zorder>
  <zorder>label_9</zorder>
  <zorder>withdrawal_time</zorder>
  <zorder>time</zorder>
  <zorder>label_13</zorder>
  <zorder>amount_money</zorder>
  <zorder>label_14</zorder>
  <zorder>withdrawal</zorder>
  <zorder>rate_of_payment</zorder>
  <zorder>pop_time</zorder>
  <zorder>time_replament</zorder>
  <zorder>label_total_percent</zorder>
  <zorder>pop_num</zorder>
  <zorder>label_15</zorder>
  <zorder>label_total_money</zorder>
  <zorder>label_total_tax</zorder>
  <zorder>label_12</zorder>
  <zorder>checkBox</zorder>
  <zorder>toolButton_menuCredit</zorder>
  <zorder>label_16</zorder>
  <zorder>toolButton_exitCredit</zorder>
  <zorder>pushButton_calculate_deposit</zorder>
  <zorder>label_17</zorder>
  <zorder>label_18</zorder>
  <zorder>label_19</zorder>
  <zorder>label_20</zorder>
  <zorder>label_21</zorder>
  <zorder>Button_pop</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
